base/number.jl

```diff
@@ -68,7 +68,9 @@ ndims(x::Number) = 0
 ndims(::Type{<:Number}) = 0
 length(x::Number) = 1
 firstindex(x::Number) = 1
+firstindex(x::Number, ::Int) = 1
 lastindex(x::Number) = 1
+lastindex(x::Number, ::Int) = 1
```

Comment:
Here as well.

Reply:
Okay

Reply:
I made the changes. When I was testing it was throwing `BoundsError` so I didn't thought of it